src/components/WebexRemoteMedia/WebexRemoteMedia.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+import WebexRemoteMedia from './WebexRemoteMedia';
+
+jest.mock('../hooks/useMeeting');
+jest.mock('../hooks/useStream');
+
+describe('Webex Remote Media component', () => {
+  describe('snapshot', () => {
+    test('matches snapshot of disabled remote audio', () => {
+      expect(shallow(<WebexRemoteMedia meetingID="remoteVideo" />)).toMatchSnapshot();
+    });
+
+    test('matches snapshot of disabled remote video', () => {
+      expect(shallow(<WebexRemoteMedia meetingID="remoteAudio" />)).toMatchSnapshot();
+    });
+
+    test('matches snapshot of disabled remote video', () => {
```

Comment:
The test name matches that of line 14. Should this be "enabled remote audio and video"?